app/assets/stylesheets/views/article-form.scss

```diff
@@ -245,9 +245,6 @@
   }
 
   &__tag-selection {
-    --tag-bg-hover: var(--base-a5);
-    --tag-bg: var(--base-a5);
```

Comment:
No longer needed - @ludwiczakpawel's [commit](https://github.com/forem/forem/commit/87ff8f16a1bfd368edbdae75ac62b687a6d50cab) went in just ahead of this and removed `--base-a5` as well as moving these `--tag` variables to a more globally available spot. Removing here means I don't override those variables with `undefined` 🙂